docker-compose.yml

```diff
@@ -0,0 +1,50 @@
+# Our entry into our docker development environment is described here.
+
+# For an overview of how to use the Docker Development Environment,
+# see `./docker/developers/README.md`
+
+# It is split among a few different files that are then merged together via
+# the following `include` section.
+
+include:
+  # Services the dashboard server depends on
+  - path: ./docker/developers/docker-compose.mysql.yml
+  - path: ./docker/developers/docker-compose.redis.yml
+
+  # MinIO S3 Emulation
```

Comment:
I don't think this needs a special grouping, its really just another service the dashboard server depends on.